src/sagemaker_inference/transformer.py

```diff
@@ -177,6 +189,8 @@ def _validate_user_module_and_set_functions(self):
             input_fn = getattr(user_module, "input_fn", None)
             predict_fn = getattr(user_module, "predict_fn", None)
             output_fn = getattr(user_module, "output_fn", None)
+            pre_model_fn = getattr(user_module, "pre_model_fn", None)
```

Comment:
Do we expect user to provide these functions?

Reply:
it's an optional function for user. if user doesn't provide `pre_model_fn` (same as `model_warmup_fn`), the default implementation for both functions will be picked.